addons/mail/static/src/components/message_tracking_value/message_tracking_value.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="mail.MessageTrackingValue" owl="1">
+        <t t-if="messageTrackingValue">
+            <div class="o_MessageTrackingValue" t-attf-class="{{ className }}" t-ref="root">
+                <span class="o_MessageTrackingValue_trackingValue">
+                    <span class="o_MessageTrackingValue_trackingValueFieldName">
+                        <t t-esc="messageTrackingValue.changedFieldAsString"/>
+                    </span>
+                    <span class="o_MessageTrackingValue_trackingValueOldValue">
+                        <t t-if="messageTrackingValue.oldValueAsString">
+                            <t t-esc="messageTrackingValue.oldValueAsString"/>
+                        </t>
+                        <i t-else="" class="mr-2">None</i>
+                    </span>
```

Comment:
All this is a single concept: the "before" part in showing a `MessageTrackingValue`. `None` is also part of it, as a fallback/`default`.

So `oldValueAsString` and fallback/default `None` should be in a field of itself.

Also name the part more briefly than `trackingValueOldValue`, into something like `before` or `beforePart`.